e2e/app/page/workspace-analysis-page.ts

```diff
@@ -107,6 +107,7 @@ export default class WorkspaceAnalysisPage extends WorkspaceBase {
     await notebookCard.selectSnowmanMenu(Option.CopyToAnotherWorkspace, {waitForNav: false});
     // Fill out modal fields.
     const copyModal = await new CopyModal(this.page);
+    await copyModal.waitForLoad();
```

Comment:
unrelated to cohort-review test. but call to `waitForLoad` function is needed to ensure modal is load & ready.